mission_control/navigator_launch/nodes/network_keepalive.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
+
+
+class KeepAlive(object):
+    '''
+    Meant to run from any number of remote computers.
```

Comment:
Well the idea is that we are testing to make sure the network link is up. If only one computer publishes it, then if that computer freezes or something, that doesn't mean the network connection dropped. Since were using tmux, if the one controller computer freezes, someone else could just attach to their session and resume control